sdks/java/io/kafka/src/test/java/org/apache/beam/sdk/io/kafka/CustomTimestampPolicyWithLimitedDelayTest.java

```diff
@@ -75,28 +82,22 @@ public void testCustomTimestampPolicyWithLimitedDelay() {
 
     // (1) Test simple case : watermark == max_timesatmp - max_delay
 
-    List<Long> input = ImmutableList.of(-200_000L,
-                                        -150_000L,
-                                        -120_000L,
-                                        -140_000L,
-                                        -100_000L,  // <<< Max timestamp
-                                        -110_000L);
+    List<Long> input =
+        ImmutableList.of(
+            -200_000L, -150_000L, -120_000L, -140_000L, -100_000L, // <<< Max timestamp
+            -110_000L);
     assertThat(getTimestampsForRecords(policy, now, input), is(input));
 
     // Watermark should be max_timestamp - maxDelay
-    assertThat(policy.getWatermark(ctx), is(now
```

Comment:
Please remove all of these spurious changes.

Reply:
Sure thing, must be some ide setting I missed.